jenkins-pipelines/ics-longevity-1tb-7days.jenkinsfile

```diff
@@ -7,7 +7,7 @@ longevityPipeline(
     backend: 'aws',
     aws_region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: '''["test-cases/longevity/longevity-1TB-7days-authorization-and-tls-ssl.yaml", "test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml"]''',
+    test_config: '''["test-cases/longevity/longevity-1TB-5days-authorization-and-tls-ssl.yaml", "test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml"]''',
```

Comment:
"test-cases/enterprise-features/ics/longevity/ics-longevity-1TB-7days-authorization-and-tls-ssl.yaml" is used, and the cs duration is still 7days (duration=10080m).

We might need to change it together?

Reply:
Not critical to change (just renaming) and don't want to test the jobs .